doc/UserGuide.md

```diff
@@ -0,0 +1,151 @@
+# User Guide
+
+## Background
+
+The Alaska Satellite Facility (ASF) has developed an advanced product toolbox (TBX) capable of conducting Interferometric SAR Processing (InSAR) with the ISCE Processoe named apt-insar. The toolbox provides automated fetching of data from ASF's datapool (https://vertex.daac.asf.alaska.edu), Digital Elevation Map (DEM) selection and stitching, and bundles NASA's Jet Propultion Labritory ISCE processor for processing interferogram products from start to finish. This allows users to begin with a pair of granule names and in one command produce an interferogram product.
+
+Insert Prargraph about InSAR here.
+
+## Output Products
+- GeoTIFF Phase image with 
+- GeoTIFF Coherence image with
+- GeoTIFF Amplitude image with
+- ArcGIS compatible ISO 19115 metadata
+
+## System Requirements
+
+* Operating System
+    - macOS 10.11 or later 
+    - Ubuntu 18.04 or later
+    - CentOS 7 or later
+* 64-bit installation
+* 16 GB of RAM
+* 20 GB of available hard disk space
+
+## Installation
+
+<details><summary>macOS</summary>
+
+1. Download the [Docker for Mac](https://download.docker.com/mac/stable/Docker.dmg) installer (~500 MB).
+
+1. Double-click Docker.dmg to open the installer, then drag Moby the whale to the Applications folder.
+
+   <p align="center"><img src="images/drag_and_drop.png" width="75%"></p>
+
+1. Double-click Docker.app in the Applications folder to start Docker, then follow any installation prompts.  When complete, "Docker Desktop is now up and running!" should be displayed.
+
+   **Note:** A Docker ID is not required.
+
+   <p align="center"><img src="images/applications_folder.png" width="75%"></p>
+   
+   <p align="center"><img src="images/docker_is_running.png" width="40%"></p>
+
+1. Click the Docker icon in the top status bar and select "Preferences".
+
+   <p align="center"><img src="images/preferences.png" width="30%"></p>
+
+   1. Select "Advanced".
+   1. Set "CPUs" to 4 or higher. More CPUs means faster processing, but may leave fewer resources for other programs during processing.
+   1. Set "Memory" to 16.0 GiB or higher.
```

Comment:
Not sure what to recommend here.  InSAR can run with 2 cores and 4 GB of memory (and maybe even less), but I assume we'll eventually merge the user guides for both RTC and InSAR?